### DISCUSSION TITLE
Challenge: toggle uBlockOrigin on and off for ALL sites with the least amount of clicks

### DISCUSSION THREAD

**Discussion by jidanni** [uBlockOrigin/uBlock-issues#1832] [Q&A] [answered]
On chrome, it takes two clicks to toggle uBlockOrigin on (or off) for the current site.
But to do so for ALL sites, four are needed. (We must disable the entire extension.)
Sure, that is a chrome problem.
But perhaps uBlockOrigin could make it easier...

**Comment by ghost** [comment, marked as answer]
Right click on the button -> Manage extension -> turn it off. This works for Chrome and Firefox.

**Reply by jidanni** [reply]
OK, we are down to three clicks.
But still takes a lot of looking in menus.

**Reply by uBlock-user** [reply]
> OK, we are down to three clicks.

Users do not keep count of clicks, so it doesn't matter.

**Comment by gorhill** [comment]
I'm fine with wholly disabling uBO taking that many clicks. Wholly disabling uBO and leaving the user unprotected should not be so easy that it could be done accidentally, without a second thought. This has been declined in the past, I won't change my mind.

**Comment by jidanni** [comment]
Well it is just like a car with only a brake pedal and no hand brake.
Sure, just put it in Park.
But it is a manual transmission car.
OK, park it on a level surface and get out the wood blocks.
